wkcuber/pyproject.toml

```diff
@@ -8,13 +8,13 @@ license = "AGPL-3.0"
 include = ["wkcuber/version.py"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.7,>=3.7.1"
```

Comment:
@jstriebel I had to make this change, because poetry complained that this requirement didn't match `webknossos/pyproject.toml` (which also uses `python = "^3.7,>=3.7.1"`). After I adapted it, I ran `poetry lock` which might have been a mistake since `wkcuber/poetry.lock` has now quite a lot of changes? Not sure whether I could have circumvented this and if yes if it's worth it to go back and remove these changes again?

What's your take on that? If we stick to the updates (which we would probably use at some point anyway), I need to fix some new linting errors. Then, the CI should turn green. Please let me know which route you prefer :)

Reply:
Ah, ok. I guess you could revert the `poetry.lock` changes and run `poetry upgrade webknossos`, which should be enough I think. Maybe we should have a separate script which cross-updates all internal dependencies we have. This could also run in CI with an assertion that nothing changes. But doesn't need to be part of this PR for sure.

Reply:
Since there was a merge conflict in the poetry.lock file, anyway, I restored the lock files from master and then only ran poetry upgrade {webknossos,wkcuber}`. The line change count is now far smaller :tada: